aspnet/migration/rc1-to-rc2.rst

```diff
@@ -172,42 +229,52 @@ The ASP.NET Core Module must be configured in ``web.config``:
+The name of the package that contains the Publish IIS tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your ``project.json`` file to inlude the ``Microsoft.AspNetCore.Server.IISIntegration.Tools`` package instead of the ``dotnet-publish-iis`` package.
 
 You can use the Publish IIS tool to publish your app with the ``web.config`` file that is required for your target environment. More information about Publish IIS changes is at https://github.com/aspnet/Announcements/issues/164.
 
-IIS integration middleware is now setup using ``WebHostBuilder`` in ``Program.Main``, and is no longer called in the ``Configure`` method of the ``Startup`` class. 
+IIS integration middleware is now setup using ``WebHostBuilder`` in ``Program.Main()``, and is no longer called in the ``Configure()`` method of the ``Startup`` class. 
 
 .. code-block:: c#
 
   public static void Main(string[] args)
   {
-    var host = new WebHostBuilder().UseIISIntegration();
+    var host = new WebHostBuilder().UseIISIntegration().Build();
   }
+  
+
+Web Deploy changes
+^^^^^^^^^^^^^^^^^^^^^^^  
 
+Delete ``RC1StarterWeb - Web Deploy-publish.ps1``. This is a script generated by VS for web deploy. There is a version for RC1 projects (dnx based) and a different script for RC2 projects (dotnet based) which are incompatible with each other. As such, when migrating to RC2, you need to delete the old script and let VS generate a new one to ensure web deploy works for the converted RC2 project.
+  
+  
+Applicationhost.config
+^^^^^^^^^^^^^^^^^^^^^^
+
+If ``applicationhost.config`` was created with RC1 or early RC2 it will point to a wrong application folder. The ``applicationhost.config`` file will read ``wwwroot`` as the application folder and this is where IIS will look for ``web.config`` file. However, since the ``web.config`` file now goes in the ``approot``, IIS won't find the file and the user may not be able to start the appliation with IIS.
+  
 
 Json configuration syntax change 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``ConfigurationRoot.ReloadOnChanged`` is no longer available, add is added explicitly via ``ConfigurationBuilder.AddJsonFile``.
-
+``ConfigurationRoot.ReloadOnChanged()`` is no longer available, add is added explicitly via ``ConfigurationBuilder.AddJsonFile()``.
 
-Changes in MVC
---------------
+Global.json
+^^^^^^^^^^^
 
-To compile views, set the ``preserveCompilationContext`` option in ``project.json`` to preserve the compilation context, as shown here:
+You must update the sdk version in ``global.json``, as this file is used to configure the solution as a whole.
 
 .. code-block:: c#  
 
   {
-    "buildOptions": {
-      "emitEntryPoint": true,
-      "preserveCompilationContext": true
-  },
+    "projects": [ "src", "test" ],
```

Comment:
The sdk version below is incorrect - it needs to be: "1.0.0-preview1-002702". The .NET Core SDK is still in preview and hasn't shipped an RC2 version yet.